Update datenshi_beatmap_criteria_en.md (#10)
osu-datenshi/website-docs: en/datenshi_beatmap_criteria_en.md

- **Q: My map hasn't been ranked but it follows the criteria?**
- A: There's a chance that your map doesn't get detected by the system, you can re-request again in the discord channel '#beatmap-requests` **Only for Donators**

-----------------------

- **Q: Why isn't my map submitted?**
- A: That can happen when : The beatmap is new, wrong beatmap link, or the beatmap database retrieval system from Datenshi to official osu! has not been updated 
- 
-----------------------

- **Q: How long does it take for my map to be ranked?**
- A: Depending on the Beatmap Nominator themselves, BN's need time to review and sort out beatmaps that follow the criteria 

-----------------------

- **Q: How many maps can I request at this time?**
- A: For now the maximum request is 4 (This might change from time to time) 

-----------------------

#### Notes
Before requesting, make sure your unranked map you are submitting are :

- In Graveyard, at least 1 month to avoid the map getting updated by the mapper (abandoned map) 
- Metadata that soothe the eyes. For example **Diao Ye Zong feat.** is bad, please avoid it. 

<h1><i class="comment icon"></i> Standard</h1>

#### Rules

> 1. The map must be playable and still follows the Bancho's RC. If not, it is likely to be loved. 

Reviewer: ```suggestion
1. Map must be playable, still following Bancho's criteria ranking. If not, it is likely to be loved.
```

Author: ```suggestion
1. Map must be playable and still follows Bancho's Ranking Criteria. If not, it is likely to be loved.
```
grammar nya yang benar kek gini. kalau pakai following gk bisa ada still nya.

> 2. A longstream map that is easily abused is likely to be loved . (Because the majority of relax players can use the DT mod to get PP)

Reviewer: ```suggestion
2. Longstream maps that are easy to abuse are likely to be loved. (Because many relax players can use the DT mod to get PP)
```

Author: ```suggestion
2. Longstream maps that are easy to abuse are likely to be loved. (Because the majority of relax players can use the DT mod to get PP)
```
pakai majority aja karena kalau pakai many kurang jelas.

> 3. Overlapped objects, 2B maps, Storyboard based maps (Maps that require the storyboard to be turned on to be played) will be loved. 

Reviewer: ```suggestion
3. Overlap objects, 2B maps, Storyboard based maps (Maps that require the storyboard to be turned on to be played) will be loved.
```

Author: ini grammar nya udah benar lho, kenapa diganti ;w;

Reviewer 2: Overlapped? Overlapping?

Author: Overlapped

Reviewer: Itu yang minta si hoshimachi suisei, dia ga ada akun github jadi gw yg wakilin

> 4. Near impossible map (Galaxy Collapse, etc) those that can be passed by a player will be ranked.

Reviewer: ```suggestion
4. Near impossible maps (Galaxy Collapse, etc.) that players can pass will be ranked.
```

> 5. A marathon map with a duration of more than 40 minutes (or those that hit the std score limit) will be loved. (Example: Songs Compilation by Nevo with 54 minutes)

Reviewer: ```suggestion
5. A map marathon with a duration of more than 40 minutes (or those that can touch the std score limit) will be loved. (Example: Nevo's 54-minute Song Compilation)
```

Author: ```suggestion
5. A marathon map with a duration of more than 40 minutes (or those that can hit the std score limit) will be loved. (Example: Nevo's 54-minute Song Compilation)
```
'map marathon' itu salah 😆 , gini aja

Reviewer 2: Until ScoreV2 enforcement is implemented, marathon map that have **drain time** longer than 40 minutes or tend to hit 2,000,000,000 or more with any kind of mod combination will be loved.

Author: Ganti begini?
```suggestion
5. Until ScoreV2 enforcement is implemented, a marathon map that have drain time longer than 40 minutes or tend to hit 2,000,000,000 or more with any kind of mod combination will be loved.
```

Reviewer: Bebas dah yg std

Author: yang atas atau bawah ciii

> 6. Abusive PP Maps (Maps that contain only 1-2 patterns, stacking longstream high bpm (relax)) are likely to be loved. 

Reviewer: ```suggestion
6. Abusive PP Maps (Maps that contain only 1-2 patterns, stacking longstream high bpm (relax)) are likely to be loved.
```

> All maps are likely to be ranked unless the map contain features mentioned in the rules above. 

Reviewer: ```suggestion
All maps are likely to be ranked unless there are features mentioned in the rules above.
```

-----------------------

- **Q: My map hasn't been ranked but it follows the criteria?**

> - A: There's a chance that your map doesn't get detected by the system, you can re-request again in discord with this format : `[REQ](osu-beatmap-link)(mode)` **Only for Donaters**

Reviewer: ```suggestion
A: There's a chance that your map doesn't get detected by the system, you can re-request again in discord channels `#beatmap-requests` **Only for Donaters**
```
ganti ini

Author: bisa diganti dengan
'you can re-request again in the discord channel #beatmap-requests'
gk pake s karena cuman satu channel

4. For rate up/speed up difficulties will be loved since it contains the same patterns as the original one.
5. Storyboard maps will be loved since it has the potential for players to cheat with dual screen.
6. SV maps that can't be sightread will be loved since it has the potential to cheat for players with dual screen.

#### 4K RULES
1. For 4K maps that have a SR>4 required a minimum of HP7 and OD7.
2. No SR manipulation is allowed (Difficulty spike) for all maps that have a SR>5

#### 4K LN 
1. For 4K maps that have a SR>8 will be pending or loved. 
2. For 4K LN maps that have a SR>5 required a minimum of HP7 and OD0.
  If it have a SR>6 with the mod DT/NC for "Easy" maps, it will not be ranked

#### 7K RULES
1. For 7K maps that have a SR>5 required a minimum of HP7 and OD7.
2. No SR manipulation is allowed (Difficulty spike) for all maps that have a SR>6

#### 7K LN 
1. For 7K maps that have a SR>10 will be pending or loved.
2. For 7K LN maps that have a SR>6 required a minimum of HP7 and OD0.
   If it have a SR>9 with the mod DT/NC for "Easy" maps, it will not be ranked

Notes:

> 1. Whether or not a pattern is good or not is a subjective matter, here I, as a mapper, have not been able to find a map with a good pattern, so please understand. Even though the map with the SR 7 is said to be playable for some people, maybe not for some people. **Except VIBRO** - Inugami Korone 

Reviewer: ```suggestion
Whether or not a pattern is good or not is a subjective matter, here I, as a mapper, have not been able to find a map with a good pattern, so please understand. Even though the map with the SR 7 is said to be playable for some people, maybe not for some people. **Except VIBRO** 
```

Author: udah gw ganti jadi lebih simple lagi, karena udah gk ngequote si william
